src/test/java/lotto/domain/LottoStoreTest.java

```diff
@@ -0,0 +1,37 @@
+package lotto.domain;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+class LottoStoreTest {
+    LottoStore lottoStore;
+
+    @BeforeEach
+    void create() {
+        List<String> list = new ArrayList<String>();
+        list.add("10, 20, 30, 40, 41, 45");
+        list.add("11, 22, 33, 40, 41, 45");
+        lottoStore = new LottoStore(list, 2);
+    }
+
+    @Test
+    @DisplayName("객체 생성")
+    void createLottoStore() {
+        Assertions.assertThat(lottoStore.getManuaCount())
+                .isEqualToComparingFieldByField(2);
+        Assertions.assertThat(lottoStore.getAutoCount())
+                .isEqualToComparingFieldByField(0);
```

Comment:
isEqualToComparingFieldByField() 테스트 메서드는 해당 객체의 인스턴스 변수 값이 같은지 비교해 줍니다.
현재 갯수를 비교하는 테스트이므로 해당 메서드 대신 다음과 같이 하는게 좋아보여요!
```suggestion
        Assertions.assertThat(lottoStore.getManuaCount()).isEqualTo(2);
        Assertions.assertThat(lottoStore.getAutoCount()).isEqualTo(0);
```